### PR TITLE
Created a Simple Layout Using CSS Grid

### PR DESCRIPTION
**Pull request by AbbasShaikh0304**

### CODE REVIEW

File: tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="header">Header</div>
```
Comment (AlishaPrasad) [inline tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html:16]: use header tag

File: tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <div class="content-large">Content #1</div>
+      <div class="content-small">Content #2</div>
+      <div class="content-small">Content #3</div>
+      <div class="content-long">Content #4</div>
+      <div class="footer">Footer</div>
```
Comment (AlishaPrasad) [inline tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html:21]: use footer tag

File: tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html (hunk 1 above)
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html]: Thank you for your suggestion. Commited the changes. @AlishaPrasad

File: tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html (hunk 2 above)
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html]: Thank you for your suggestion. Commited the changes. @AlishaPrasad

File: tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/style.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+body {
```
Comment (Tanvir04khan) [inline tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/style.css:1]:
```suggestion
* {
margin: 0;
padding:0;
box-sizing: border-box;
}
```

**Review by coder-mezab — APPROVED** [review]

**Review by shaikh-7abish — APPROVED** [review]

**Review by nasir-k — APPROVED** [review]

File: tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/style.css (hunk 3 above)
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/style.css]: Taken your suggestion in account. Please check commits @Tanvir04khan

**Pull request merged**